src/compose/ComposeText.js

```diff
@@ -68,8 +68,14 @@ class ComposeText extends React.Component {
     } else if (isTopicNarrow(narrow)) {
       sendMessage(auth, 'stream', narrow[0].operand, narrow[1].operand, text);
     }
+    this.clearInput();
```

Comment:
clearInput() has two parts, one that clears the TextInput and other that clears the state's "text". IMO grouping them together would be a good idea since if needed can be easily reused. Thoughts?